Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,174 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+using System.IO;
+using System.Runtime.ExceptionServices;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
         /// <summary>
         /// Return name and balance(current) of person who had the biggest historic balance.
         /// </summary>
-        public static string FindHighestBalanceEver(string[] peopleAndBalances)
+
+        private const string NotFound = "not found";
+
+        public static string FindHighestBalanceEver(string balancesFile)
+        {
+            string[] peopleAndBalances = ReturnFileContent(balancesFile);
```

Comment:
Should I reader care that you return file content? Or should the reader care that you ReadPeopleAndBalances?